#!/usr/bin/env python2.7

PACKAGE='hanse_sonarlocalization'
import roslib; roslib.load_manifest(PACKAGE)
from math import pi
from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("particle_count", int_t, 0, "particleCount", 200, 10, 10000)
gen.add("perturb_position", double_t, 0, "perturbPosition", 0.05, 0, 10)
gen.add("perturb_rotation", double_t, 0, "perturbRotation", 5 * pi / 180, 0, 2 * pi)
gen.add("weight_sigma", double_t, 0, "weightSigma", 10, 0.1, 100)

exit(gen.generate(PACKAGE, "sonarlocalization", "ParticleFilter"))
